src/python/WMCore/Storage/DeleteMgr.py

```diff
@@ -84,43 +85,52 @@ def initialiseSiteConf(self):
         Extract required information from site conf and TFC
 
         """
-
-        implName = self.siteCfg.localStageOut.get("command", None)
-        if implName == None:
-            msg = "Unable to retrieve local stage out command\n"
-            msg += "From site config file.\n"
-            msg += "Unable to perform StageOut operation"
-            raise StageOutInitError(msg)
-        msg = "Local Stage Out Implementation to be used is:"
-        msg += "%s\n" % implName
-
-        pnn = self.siteCfg.localStageOut.get("phedex-node", None)
-        if pnn == None:
-            msg = "Unable to retrieve local stage out phedex-node\n"
-            msg += "From site config file.\n"
-            msg += "Unable to perform StageOut operation"
-            raise StageOutInitError(msg)
-        msg += "Local Stage Out PNN to be used is %s\n" % pnn
-        catalog = self.siteCfg.localStageOut.get("catalog", None)
-        if catalog == None:
-            msg = "Unable to retrieve local stage out catalog\n"
-            msg += "From site config file.\n"
-            msg += "Unable to perform StageOut operation"
-            raise StageOutInitError(msg)
-        msg += "Local Stage Out Catalog to be used is %s\n" % catalog
-
-        try:
-            self.tfc = self.siteCfg.trivialFileCatalog()
-            msg += "Trivial File Catalog has been loaded:\n"
-            msg += str(self.tfc)
-        except Exception as ex:
-            msg = "Unable to load Trivial File Catalog:\n"
-            msg += "Local stage out will not be attempted\n"
-            msg += str(ex)
+        
+        self.stageOuts = self.siteCfg.stageOuts
+
+        msg = ""
+        msg += "There are %s stage out definitions.\n" % len(self.stageOuts)
+        
+        for stageOut in self.stageOuts:
+            msg = ""
+            for k in ['phedex-node','command','storageSite','volume','protocol']:
+                v = stageOut.get(k)
+                if v is None:
+                    msg += "Unable to retrieve "+k+" of this stageOut: \n"
+                    msg += stageOutStr(stageOut) + "\n"
+                    msg += "From site config file.\n"
+                    msg += "Continue to the next stageOut.\n"
+                    logging.info(msg)
+                    continue
+
+            storageSite = stageOut.get("storageSite")
+            volume = stageOut.get("volume")
+            protocol = stageOut.get("protocol")
+            command = stageOut.get("command")
+            pnn = stageOut.get("phedex-node")
```

Comment:
Yes, we can keep the old terminology for now since I am not sure whether it is used somewhere else outside of WMCore/Storage. After things are running we can attempt to change the terminology